provider/cluster_rosa_classic_resource.go

```diff
@@ -686,7 +686,7 @@ func createClassicClusterObject(ctx context.Context,
 			return nil, errors.New(errHeadline + "\n" + description)
 		}
 		vBuilder := cmv1.NewVersion()
-		versionID := state.Version.Value
+		versionID := fmt.Sprintf("openshift-v%s", state.Version.Value)
```

Comment:
I think better moving the format string to a const

Reply:
Please notice that here removing the prefix from `state.Version.Value` and in the next comment removing it from `CurrentVersion.Value`
I don't think const is required here